src/sass/dashboard/_allUploadStats.scss

```diff
@@ -1,12 +1,9 @@
 .uploadStats {
-  border: 1px solid $dblue;
+  border: 1px solid $primary-blue;
   border-radius: 10px;
   margin: 3rem;
   padding: 1rem;
-  color: $dblue;
+  color: $primary-blue;
   text-align: center;
   font-size: 1.5rem;
```

Comment:
Will need to increase font-size a bit.
![image](https://user-images.githubusercontent.com/22422113/53752411-63e2c080-3e63-11e9-8cbe-310dfe61025c.png)

Reply:
I will address this issue in _SCSS_ for now.